include/manager.hpp

```diff
@@ -182,6 +182,13 @@ class Manager
     void SetTimerToDetectVpdCollectionStatus();
 #endif
 
+    /**
+     * @brief An API to perform backup and restore VPD.
+     *
+     * API used to invoke backup and restore VPD keyword's.
```

Comment:
Don't have to add this if there is no substantial information to put.

Reply:
Removed.